src/storage/types.ts

```diff
@@ -1,9 +1,56 @@
+// eslint-disable-next-line import/no-cycle
 import { Sequence, StoredAnswer } from '../store/types';
 
+/**
+The ParticipantData is a JSON object that contains all of the data for all of the participants in your study. It is structured as a list. Each element refers to a participants data or a configuration. While in many cases there is only one configuration per study, the study creator is allowed to change the configuration file after the study has already been completed by other participants. The data for each participant will have a "participantConfigHash" which refers to a particular configuration which is also in this list.
+
+Below we have an example of a participants data.
+``` JSON
+[
+    {
+        "participantId": <UUID4>,
+        "participantConfigHash": <CONFIG_ID>,
+        "sequence": {
+            ...
+        },
+        "answers": {
+            ...
+        },
+        "searchParameters": {
+            ...
+        }
+    }
+]
+```
+Each key in answer will be labeled the same as the response component that it refers to. The sequence shows the order that the participant saw each component (since these may be different for each participant if the configuration sequence has some randomization). This answer wil contain information such as the start time, the end time, and all of the window events. See the example below.
```

Comment:
```suggestion
Each key in answer will be labeled the same as the response component that it refers to. The sequence shows the order that the participant saw each component (since these may be different for each participant if the configuration sequence has some randomization). This answer will contain information such as the start time, the end time, and all of the window events. See the example below.
```